Makefile

```diff
@@ -34,4 +34,4 @@ snapshot:
 	docker exec -it edx.pubfe npm run snapshot
 
 lint:
-	docker exec -it edx.pubfe npm run lint && npm run stylelint
+	docker exec -it edx.pubfe npm run lint && docker exec -it edx.pubfe npm run stylelint
```

Comment:
Nit: I think this is a cleaner way of making this change so we don't have to copy the docker exec -it edx.pubfe
`docker exec -it edx.pubfe bash -c 'npm run lint && npm run stylelint'`